chia/full_node/full_node_api.py

```diff
@@ -247,9 +248,12 @@ async def respond_transaction(
             self.full_node.full_node_store.peers_with_tx.pop(spend_name)
 
         # TODO: Use fee in priority calculation, to prioritize high fee TXs
-        await self.full_node.transaction_queue.put(
-            TransactionQueueEntry(tx.transaction, tx_bytes, spend_name, peer, test)
-        )
+        try:
+            await self.full_node.transaction_queue.put(
+                TransactionQueueEntry(tx.transaction, tx_bytes, spend_name, peer, test), peer.peer_node_id
+            )
+        except TransactionQueueFull:
+            pass  # we can't do anything here, the tx will be dropped. We might do something in the future.
```

Comment:
right, I see. peers aren't sending these TXs to us, we're requesting them. In the future we might want to stop requesting more TXs once we get close to the limit

Reply:
i could add that

Reply:
that should be a separate PR. we don't want to tie too many independent changes together